src/Shared/UnitTests/FileMatcher_Tests.cs

```diff
@@ -1373,6 +1372,49 @@ public void FileEnumerationCacheTakesExcludesIntoAccount()
             }
         }
 
+        [Fact]
+        public void DriveEnumerationWildcardThrowsException()
```

Comment:
Name seems wrong; should this be something like

```suggestion
        public void DriveEnumerationWildcardIsObserved()
```

?